publish/publish.proj

```diff
@@ -65,6 +65,14 @@
                    Condition="'@(_MissingBlobNames)' == ''" />
     <Message Importance="High" Condition="'@(_MissingBlobNames)' != ''" Text="Skipping finalization, missing these blobs: '@(_MissingBlobNames)'" />
   </Target>
+  
+  <Target Name="UpdateVersionsRepo"
```

Comment:
Any reason we wouldn't do this in the publish build definition like in corefx? https://github.com/dotnet/corefx/blob/b5ac646f30e2d36729bdb95dac072ceb1e5b186d/buildpipeline/DotNet-Trusted-Publish.json#L177